README.md

```diff
@@ -28,39 +26,45 @@ remotes::install_github("pharmaverse/pharmaversesdtm", ref = "devel")
+* Following [best practice](https://r-pkgs.org/data.html#sec-data-data), each dataset is stored as a `.rda` file whose name is consistent with the name of the dataset, e.g., dataset `xx` is stored as `xx.rda`. The easiest way to achieve this is to use `usethis::use_data(xx)`
+* The programs in `data-raw/` are stored within the `{pharmaversesdtm}` GitHub repository, but they are **not** part of the `{pharmaversesdtm}` package--the `data-raw/` folder is specified in `.Rbuildignore`.
+* When you run a program that is in the `data-raw/` folder, you generate a dataset that is written to the `data/` folder, which will become part of the `{pharmaversesdtm}` package.
+* The names and sources of test datasets are specified in `R/data.R`, for the purpose of generating documentation in the `man/` folder.
 
 ## Adding New SDTM Datasets
 
-* Create a program in the `dev/` folder, named `<name>.R`, where `<name>` is the SDTM domain name, (e.g. `rs.R`), to generate the test data and output `<name>.rda` to the `data/` folder. Use CDISC pilot data such as `dm` as input in this program in order to create realistic synthetic data that remains consistent with other domains. Note that __no personal data should be used__ as part of this package, even if anonymized.
+* Create a program in the `data-raw/` folder, named `<name>.R`, where `<name>` should follow the [naming convention](#naming), to generate the test data and output `<name>.rda` to the `data/` folder.
+    * Use CDISC pilot data such as `dm` as input in this program in order to create realistic synthetic data that remains consistent with other domains (not mandatory).
+    * Note that **no personal data should be used** as part of this package, even if anonymized.
 * Run the program.
 * Reflect this update, by specifying `<name>` in `R/data.R`.
 * Run `devtools::document()` in order to update `NAMESPACE` and update the `.Rd` files in `man/`.
+* Add your GitHub handle to `.github/CODEOWNERS`
+* Update `NEWS.md`
 
 ## Updating Existing SDTM Datasets
 
-* Rename the source dataset as `raw_<name>`, where `<name>` is the SDTM domain name (e.g. rename `ds` to `raw_ds`), and then save it to the `data/` folder as `raw_<name>.rda` (e.g. `save(raw_ds, file = "data/raw_ds.rda")`).
-* Create a program in the `dev/` folder, named `update_<name>.R`, to load `raw_<name>.rda`, make the updates, and output `<name>.rda` to the `data/` folder.
-* Run the program.
-* Reflect this update, by specifying both `raw_<name>` and `<name>` in `R/data.R`.
+* Locate the existing program `<name>.R` in the `data-raw/` folder, update it accordingly. 
+* Run the program, and output updated `<name>.rda` to the `data/` folder.
 * Run `devtools::document()` in order to update `NAMESPACE` and update the `.Rd` files in `man/`.
+* Add your GitHub handle to `.github/CODEOWNERS`
```

Comment:
full stops are missing at the end of the last two bullet points.

Reply:
updated